management/prompt.go

```diff
@@ -12,6 +14,10 @@ type PromptManager struct {
 	*Management
 }
 
+type promptCustomText struct {
+	Text *string
```

Comment:
Should we keep this as `*string` or perhaps a `map[string]interface{}`?

Considering the payloads will be of format: `{ "login": { "title": "Welcome" } }`

https://auth0.com/docs/brand-and-customize/text-customization-new-universal-login